core/workspace.js

```diff
@@ -178,6 +178,20 @@ Blockly.Workspace.prototype.getAllBlocks = function() {
 };
 
 /**
+ * Checks whether all inputs are filled.
+ * @return {boolean} True if all inputs are filled.
+ */
+Blockly.Workspace.prototype.isFilled = function() {
```

Comment:
This function's name is ambiguous--from the PR title I thought this was about rendering/fill colors, not connections.

How about allInputsConnected or allInputsFilled?